modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMHomePage.path

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+<title>PRMHomePage</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">PRMHomePage</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>CLICK_MDF</td>
+	<td>//div[contains(@class,'structure-item-container')]//a[contains(text(), 'MDF')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @juliosilveira77, I think that this path isn't secure, because this class isn't something that reflects what the button really is. Could you please see [how I did here](https://github.com/AfonsoHSantos/liferay-portal/blob/master/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMarketing.path) and do the same here? Thanks!